Components/sections/HomePage/Clients.jsx

```diff
@@ -25,11 +25,11 @@ export default function Clients() {
               </div>
             </div>
 
-            <div className=" flex flex-col py-20 w-[80%] mt-auto h-fit pr-10 pl-[3.75rem] space-y-6 text-white bg-[url('/assets/images/noise.webp')] bg-[#223240] rounded-tr-[60%] rounded-tl-[60%] rounded-bl-[80%]">
+            <div className=" flex flex-col py-20 w-[80%] mt-auto h-fit pr-10 pl-[3.75rem] space-y-4 text-white bg-[url('/assets/images/noiseonclients.webp')] bg-[#223240] rounded-tr-[60%] rounded-tl-[60%] rounded-bl-[80%]">
```

Comment:
`bg-[url('/assets/images/noiseonclients.webp')]` and `rounded-tl-[60%]` can be set in config like , then all the colors , try to add them in config

```javascript
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    "./app/**/*.{js,ts,jsx,tsx}",
    "./pages/**/*.{js,ts,jsx,tsx}",
    "./components/**/*.{js,ts,jsx,tsx}",
  ],

theme: {
    extend: {
      borderRadius: {
        "8/10": "80%",
        "1/1": "100%",
        "5/10": "50%",
        "6/10": "60%",
      },
        backgroundImage: {
        grain: "url('/assets/images/noiseonclients.webp');",
    }
   }
  }
}

```